COVID19/src/screens/main.js

```diff
@@ -17,6 +19,38 @@ const Item = ({item, onPress}) => (
 );
 
 const Main = ({navigation}) => {
+  const signOutReadAsyncToken = async () => {
+    try {
+      const token = await AsyncStorage.getItem(constans.asyncTokenVar);
+      if (token !== null) {
+        const config = {
+            headers: { Authorization: 'Bearer ' + token}
+        };
+        const bodyParameters = {
+          key: ''
+        };
+        axios.delete( 
+          constans.urlSingOut,
+          bodyParameters,
+          config
+        ).then((res) => {
+          console.log(res.data.code);
+          if (res.data.code === constans.signOutResDataCode)
+          {
+            navigation.navigate('Login');
+          }
+          else{
+            alert(constans.errorSignOut);
+          }  
+        }).catch((e) => {
+          alert(e.toString());
+        });
```

Comment:
can you create a services.js file and add the axios delete function?
example
```
export default {
  signIn: async (email, password) => {
    return axios.post(`URL`, {
      email,
      password,
    });
  },
};

```

Reply:
yes, good idea.